WordPress/Classes/ViewRelated/Reader/DetailWebView/ReaderCSS.swift

```diff
@@ -21,8 +21,8 @@ struct ReaderCSS {
     ///
```

Comment:
The comments reference a 2 day update period but we've changed to 5. What about changing to reference the `expirationDays` variable?